service/version_bundles.go

```diff
@@ -158,6 +158,12 @@ func NewVersionBundles() []versionbundle.Bundle {
 		},
 		{
 			Changelogs: []versionbundle.Changelog{
+				{
+
+					Component:   "containerlinux",
+					Description: "Updated containerlinux version to 1576.5.0.",
+					Kind:        "changed",
```

Comment:
Lets maybe use the versionbundle types here. See https://godoc.org/github.com/giantswarm/versionbundle#pkg-constants.

Reply:
oh jesus, github didn't show your comment until i clicked on merge :(

Reply:
Yeah, Github is a bitch when you keep tabs open. They are not really reliably updating the user's view. No worries. I will boyscout this at some point because this is not the only place where the constants are not used.